Code/Editor/Core/EditorActionsHandler.cpp

```diff
@@ -1778,6 +1833,13 @@ void EditorActionsHandler::OnOnlyShowHelpersForSelectedEntitiesChanged([[maybe_u
     m_actionManagerInterface->TriggerActionUpdater(OnlyShowHelpersForSelectedEntitiesIdentifier);
 }
 
+/*
+void EditorActionsHandler::OnViewportInfoDisplayStateChanged([[maybe_unused]] AZ::AtomBridge::ViewportInfoDisplayState state)
+{
+    m_actionManagerInterface->TriggerActionUpdater(ViewportDisplayInfoStateChangedUpdaterIdentifier);
+}
+*/
+
```

Comment:
Is this meant to be commented out here?

Reply:
No, I had deleted it but forgot to push the change. Fixed!